src/components/quiz/MultipleChoice.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMemo } from "react";
+
+import { MultipleChoiceProps } from "@/apis/quiz/quiz.type";
```

Comment:
아래와 같이 표현해주는 건 지금 코드와 어떻게 다를까요?
```suggestion
import type { MultipleChoiceProps } from "@/apis/quiz/quiz.type";
```